hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockReconstructedStripeInputStream.java

```diff
@@ -118,6 +129,10 @@ public ECBlockReconstructedStripeInputStream(ECReplicationConfig repConfig,
     // The EC decoder needs an array data+parity long, with missing or not
     // needed indexes set to null.
     decoderInputBuffers = new ByteBuffer[getRepConfig().getRequiredNodes()];
+
+    ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat(
+        "ec-reader-for-" + blockInfo.getBlockID() + "-%d").build();
```

Comment:
Can we just explicitly say -TID-%d, just not confuse with block indexes.

Reply:
I'm not sure what you mean here?